### PR TITLE
Adds simple homepage layout

### LINKED ISSUE #47: Homepage mock-up
Views in html to see the ux and decide on web page's structure

### PR DESCRIPTION
**Pull request by gosiamaks**
Closes #47, connected to #49

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h1>Wanna have lunch?</h1>
+<button>YES!</button>
+<h1>What do you have in your fridge?</h1>
+
```
Comment (basiacierocka) [inline app/views/home/index.html.erb]: delete empty lines

File: spec/controllers/home_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+RSpec.describe HomeController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
```
Comment (basiacierocka) [inline spec/controllers/home_controller_spec.rb]: remove empty line

File: spec/helpers/home_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# describe HomeHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe HomeHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (CallMeSandra) [inline spec/helpers/home_helper_spec.rb]: It doesn't do anything, delete it

File: spec/views/home/index.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "home/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (CallMeSandra) [inline spec/views/home/index.html.erb_spec.rb]: Also doesn't do anything

File: spec/controllers/home_controller_spec.rb (hunk 2 above)
Comment (CallMeSandra) [inline spec/controllers/home_controller_spec.rb]: I think this empty line is useless

File: app/views/home/index.html.erb (hunk 1 above)
Comment (CallMeSandra) [inline app/views/home/index.html.erb]: empty line

File: spec/controllers/home_controller_spec.rb (hunk 2 above)
Comment (basiacierocka) [inline spec/controllers/home_controller_spec.rb]: remove empty line

Comment (basiacierocka) [inline spec/controllers/home_controller_spec.rb]: at 11

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,14 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LunchApp</title>
+    <title>Luncher</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
```
Comment (basiacierocka) [inline app/views/layouts/application.html.erb]: remove empty line 10

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,14 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>LunchApp</title>
+    <title>Luncher</title>
     <%= csrf_meta_tags %>
```
Comment (CallMeSandra) [inline app/views/layouts/application.html.erb]: empty line

**Comment by ghost** [comment]
for me looks ok

File: app/assets/javascripts/home.coffee [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (gotar) [inline app/assets/javascripts/home.coffee]: you can drop this file if you do not use it. No reason to keep empty one

File: app/helpers/home_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
Comment (gotar) [inline app/helpers/home_helper.rb]: you don't need those file

**Comment by gotar** [comment]
ok you can merge but before squeeze commits to single one (manually)

**Pull request merged**